Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py

```diff
@@ -1287,8 +2435,16 @@ def get_indicators(client: Client, **kwargs):
         Returns filtered indicators by parameters from ThreatStream.
         By default the limit of indicators result is set to 20.
     """
-    limit = kwargs['limit'] = int(kwargs.get('limit', 20))
+    page = kwargs.get('page')
+    page_size = kwargs.get('page_size')
     offset = kwargs['offset'] = 0
+    limit = kwargs['limit'] = int(kwargs.get('limit', 20))
+    params = return_params_of_pagination_or_limit(arg_to_number(page), arg_to_number(page_size), arg_to_number(limit))
+    kwargs.update(params)
+    if 'page' in kwargs:
```

Comment:
Not sure why are you doing this.
Just pass the arguments as is

Reply:
We don't want to pass the page and page_size parameters in the request. (kargs passed to the request as is)